app/views/events/show.html.slim

```diff
@@ -2,17 +2,17 @@
   .card-header
     | Event
   .card-body
-    h4.card-title = @event.name
-    p.card-text = @event.content
-    - if !@event.tickets.empty?
+    h4.card-title = @view.event.name
+    p.card-text = @view.event.content
+    - if !@view.event.tickets.empty?
```

Comment:
`- if !@view.event.tickets.empty?`
można zastąpić
`- unless @view.event.tickets.empty?`
albo
`- if @view.event.tickets.any?`

moim zdaniem trochę czytelniejsze niż `if !`, ale to kwestia preferencji